### ISSUE TITLE
Missing ViewPorts and VP Tools Documentation?

### ISSUE THREAD

**Issue by JMarshQPK** [gtalarico/pyrevitplus#33]
I have two groups of tools called "ViewPorts" and "VP Tools" under my pyRevitPlus tab. I'm getting an error and a tool is not working as I expected, so I'm looking for documentation and/or the developer of those specific tools.
Thanks.

**Comment by ejs-ejs** [comment]
https://github.com/ejs-ejs/py-ejs
Sorry, not enough time right now to make it a proper extension.

**Comment by JMarshQPK** [comment]
No, problem. Good to know that is the case. If anyone reading this has time to pick it up, I'm interested in understanding its intent. (If I'm correct about its intent, it could save hours per project easily.)

**Comment by ejs-ejs** [comment]
The primary intent was to ease the placement of views and schedules on different sheets: compose a sheet, save locations and apply them to a view on a different sheet. The drawback: a single position can be saved right now, so aligning viewports, schedules and legends will take some time; but less if it had to be done manually.
The grid / level tools are meant to ease the alignment of these elements on different views: copy the settings (endpoint locations in a view, bubbles) from one view and apply them to the other.
Align tools are meant for tags and spot elevations: select an object in the source position, and the tool will alter X or Y coordinate for the next ones. The spot elevations can be aligned by vertical axis only, but they were the pain to be aligned in the drawings.

**Comment by JMarshQPK** [comment]
Thanks for the info! I am primarily interested in the grid/level tools which sound like they function exactly how I was envisioning.
My last building had around 180 grids and there were at least 10 overall floor plans, so that functionality would have been great. Hopefully one day there is time to dive back into it!

**Comment by ejs-ejs** [comment]
Just try copying the Github files to %APPDATA%\pyRevit\Extensions\Py-ejs.extension folder. I'm using this collection as-is, but a small changes to the pyRevit core libraries may be needed.

My motivation to write these tools was an 8-storey building with c.a. 120 sheets of drawings.
